examples/ge/ge_df.cc

```diff
@@ -11,16 +11,18 @@
 #include <execution>
 #define HAS_EXECUTION_HEADER
 #endif
-#include "blockmatrix.h"
+#include "../blockmatrix.h"
 
 // #include <omp.h> //
 
 #include "GE/GEIterativeKernelDF.h"          // contains the iterative kernel
 //#include "GE/GERecursiveParallelKernel.h"  // contains the recursive but serial kernels
 #include "GE/GERecursiveSerialKernel.h"    // contains the recursive and parallel kernels
 
-#include TTG_RUNTIME_H
-IMPORT_TTG_RUNTIME_NS
+#include "ttg.h"
+using namespace ttg;
+
+#include <madness/world/world.h>
```

Comment:
Why do we need the madness world header here?

Reply:
This is required for using hashT from the madness namespace (for hashing pair of pairs) which is implemented in madness-src and also for using xterm_debug.